content/administer-tugboat-crew/account-authentication.md

```diff
@@ -0,0 +1,359 @@
+
+Use a two-factor authentication app on your mobile device to scan the QR code, or manually enter the code into your
+preferred 2FA tool.
+
+![View of the Two-Factor Authentication dialog showing the QR code and code manually listed](../../_images/scan-new-qr-code-or-manually-enter-it.png)
+
+Enter the 6-digit code from your two-factor authentication app into the dialog box, and press the Verify button.
+
+![6-digit code typed into dialog box, with arrow pointing to the Verify button](../../_images/enter-new-code-and-press-verify-button.png)
+
+You'll see a list of recovery codes for your account. You can use the {{% ui-text %}}Copy to Clipboard{{% /ui-text %}}
+link to copy the codes. **Make sure you save these in a safe place, as they will not be displayed again.** Press the
+{{% ui-text %}}Close{{% /ui-text %}} button once you've saved the codes.
+
+![List of Recovery Codes with arrows pointing to the "Copy to Clipboard" UI element and the "Close" button](../../_images/copy-codes-to-clipboard-and-press-close.png)
+
+{{% /expand%}}
+
+## Renew two-factor authentication codes
+
+If you forgot to save your two-factor authentication codes, or have used them all and want new ones, you can use Renew
+to generate new codes.
+
+To renew two-factor authentication codes for your Tugboat account:
+
+1. Click the {{% ui-text %}}User{{% /ui-text %}} drop-down in the upper right-hand corner of the Tugboat dashboard, and
+   select {{% ui-text %}}Profile{{% /ui-text %}}.
+2. Go to the {{% ui-text %}}Two-Factor Authentication{{% /ui-text %}} section of the User Profile, and click
+   {{% ui-text %}}Renew{{% /ui-text %}}.
+3. Open your two-factor authentication app, and enter the 6-digit code from it into the Tugboat dialog box.
+4. Enter the 6-digit code from your two-factor authentication app into the dialog box, and press the Verify button.
+5. You'll see a new list of recovery codes for your account. You can use the {{% ui-text %}}Copy to
+   Clipboard{{% /ui-text %}} link to copy the codes. **Make sure you save these in a safe place, as they will not be
+   displayed again.** Press the {{% ui-text %}}Close{{% /ui-text %}} button once you've saved the codes.
+
+{{%expand "Visual Walkthrough" %}}
+
+Click the {{% ui-text %}}User{{% /ui-text %}} drop-down in the upper right-hand corner of the Tugboat dashboard, and
+select {{% ui-text %}}Profile{{% /ui-text %}}.
+
+![Tugboat Dashboard view with arrow pointing to User drop-down, and then Profile](../../_images/go-to-user-select-profile.png)
+
+Go to the {{% ui-text %}}Two-Factor Authentication{{% /ui-text %}} section of the User Profile, and click
+{{% ui-text %}}Renew{{% /ui-text %}}.
+
+![Tugboat's User Profile pane with Two-Factor Authentication section circled and an arrow to the Renew link](../../_images/go-two-factor-authentication-click-renew.png)
+
+Use a two-factor authentication app on your mobile device to scan the QR code, or manually enter the code into your
```

Comment:
The QR code is not displayed when renewing recovery codes. This step should be removed